src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -573,6 +570,25 @@ const generateTaskRole = (frameworkName, taskRole, jobInfo, frameworkEnvList, co
       name: `${encodeName(frameworkName)}-regcred`,
     });
   }
+  // add storages
+  if ('extras' in config && config.extras.storages) {
+    for (let storage of config.extras.storages) {
```

Comment:
And what if storage name is missing. Will use default storage?

Reply:
I thought "default storage" is only used in old config `teamwise_storage`, and current backward compatibility has supported it. Do we still need "default storage" in new design?

Some users do not need storage, and adding default storage may break their job when there're network issues etc. with storage; For other users, they should now the storage explicitly if they need to use.

Reply:
I mean do we support spec such as:
```yaml
extras:
  storages: default
```

Let customer does't care about the storage, and just use the default one.
If `storage` field is missing, it means user doesn't want storage.

Since `NNI` has such require, and it doesn't have permission to get every groups storage config. Add the `default` option will let third-party tools leverage the storage feature. And `runtime plugin` can be deprecated.

Reply:
updated, set `storages: []` to use default storages

Reply:
> Since `NNI` has such require, and it doesn't have permission to get every groups storage config. Add the `default` option will let third-party tools leverage the storage feature. And `runtime plugin` can be deprecated.

A basic question, how (who) to specify a storage as default (for user)?

Reply:
storage is configured for each group (not user, user belongs to one/multi group)

**admin** specifies default storage when he configured storage for groups (create in deployment config or update through group api)

Reply:
Got it, thanks. Thus *admin* could assign a storage as *default* for a group, right? 
1. If a user belongs to multiple groups, do we allow multiple default storages (one for each group)?
1. If we want to leverage the default storage (e.g. in Jupyter extension, VSC extension), could the sdk be aware which storage is default? (if we could, we could reduce user's effort to select)

Reply:
yes, 1. yes, 2, yes, currently the first storage (in array) in group config is default storage

`/api/storages` will return storage list which current user has permission to, we could also add a `default` query in this api to return default storage list for current uesr.

Reply:
added `default` query in `/api/storages`